about.html

```diff
@@ -51,69 +51,89 @@ <h1 class="margin-bottom large-title">About</h1>
           <ul>
-            <li>Research and Development by academics, researchers and enthusiasts.</li>
+            <li>Allowing end users to have a reliable source of OpenJDK binaries for all platforms.</li>
+            <li>Research and Development by OpenJDK developers, academics and researchers.</li>
             <li>An open, common, audited, build infrastructure for vendors to use (if they wish).</li>
-            <li>Providing binaries for bleeding edge early adopters.</li>
-            <li>Providing a place to try out build infrastructure ideas that might one day be moved to OpenJDK proper.</li>
+            <li>Providing a place to try out build infrastructure ideas that might one day be re-implemented in OpenJDK proper.</li>
           </ul>
         </p>
+      </div>
 
+      <div class="margin-bottom">
+        <h2 class="bold">I'm a Java / OpenJDK User</h2>
         <p>
-          Although we will be making the binaries to the highest quality possible via jtreg, jcov test coverage and possibly other donated test
-          suites. It is <strong>not</strong> the intention that the binaries produced will be professionally supported. See
-          <a href="./support.html">Support</a> for details.
+          We will be providing solidly made binaries to the highest quality possible utilising existing OpenJDK test suites
+          (via jtreg, jcov test coverage) and possibly other OSS donated test suites. However, these binaries are
+          <strong>not</strong> professionally supported.  See <a href="./support.html">Support</a> for details.
         </p>
 
+        <h3 class="bold">Supported Platforms</h3>
         <p>
-          There may be some areas of overlap with other past/present efforts, but that's OK.  We expect any duplication to converge in due course and have
-          started working with other platform specific OpenJDK build efforts.
+          We currently have Linux x64 and Mac OS X and are looking to rapidly add the other common Linux variants + Windows and
+          Solaris.  We envisage some strong collaboration can form here with IcedTea and other specific platform efforts!
         </p>
 
-        <p>
-          Discussions will take place on the <a href="http://mail.openjdk.java.net/mailman/listinfo/adoption-discuss">Mailing List</a>, or via
-          the <a href="https://github.com/AdoptOpenJDK/openjdk-build/issues">Github issues</a>; and consensus recorded on this project website or the appropriate GitHub repo wiki as appropriate.
-        </p>
       </div>
 
       <div class="margin-bottom">
-        <h2 class="bold">Supported Forests</h2>
+        <h2 class="bold">I'm an OpenJDK Developer</h2>
+
+        <p>
+          We anticipate that this effort will lead to an easier way for OpenJDK developers to submit, review and test patches for
+          multiple platforms <strong>before</strong> submitting to OpenJDK proper.
```

Comment:
Consider changing "OpenJDK proper" to "the [OpenJDK] project(http://openjdk.java.net/)".